src/main/java/com/iexec/worker/tee/scone/SconeConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.iexec.worker.tee.scone;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SconeConfig {
+
+    String sconeCasAddress;
```

Comment:
Can you add very few words (in code comments) explaining what's CAS and LAS are?

Reply:
Sure, it isn't clear in fact.